Exec/gravity_tests/DustCollapse/bc_fill_nd.F90

```diff
@@ -47,9 +45,9 @@ subroutine ca_hypfill(adv, adv_lo, adv_hi, &
     end if
 
     ! XLO
-    if ( bc(1,1,1) == FOEXTRAP .and. adv_lo(1) < domlo(1)) then
-       do k = adv_lo(3), adv_hi(3)
-          do j = adv_lo(2), adv_hi(2)
+    if ( bc(1,1,1) == FOEXTRAP .and. lo(1) < domlo(1)) then
+       do k = lo(3), hi(3)
```

Comment:
does this change mean that you are not filling corners?

Reply:
It's not a functional change == `lo == adv_lo`, `hi == adv_hi`.

Reply:
The general direction we've been moving towards is that routines operate on (lo, hi) always. I could add a comment to this effect but I wonder if the issue is more broadly that we need better education/documentation on this new strategy?